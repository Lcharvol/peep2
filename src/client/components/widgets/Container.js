import styled from 'styled-components';

const Container = styled.div`
  display:flex;
  flex-direction:column;
  position:relative;
  padding: 20px;
  width:95%;
  height: 100vh;
  margin:auto;
  margin-top:25px;
  margin-bottom:25px;
  background-color: #394b59;
  border-radius: 2px;
`;

export default Container;
