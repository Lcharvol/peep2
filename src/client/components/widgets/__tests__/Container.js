import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme'; // eslint-disable-line
import Container from '../Container';

describe('<Container />', () => {
  test('should render a <Container />', () => {
    const tree = shallow(<Container />);
    expect(tree).toMatchSnapshot();
  });
  test('should <Container /> be flex', () => {
    const tree = shallow(<Container />);
    expect(tree).toHaveStyleRule('display', 'flex');
  });
});
