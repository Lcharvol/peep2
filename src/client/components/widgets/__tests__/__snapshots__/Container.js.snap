// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render a <Container /> 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa hwASSu"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa hwASSu"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <styled.div />,
    "_instance": StyledComponent {
      "attrs": Object {},
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "generatedClassName": "hwASSu",
        "theme": null,
      },
      "unsubscribeId": -1,
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": Object {
      "generatedClassName": "hwASSu",
      "theme": null,
    },
    "_rendered": <div
      className="sc-bdVaJa hwASSu"
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;
